spring-rest/src/main/java/net/baeldung/services/ExampleService.java

```diff
@@ -0,0 +1,12 @@
+package net.baeldung.services;
+
+import net.baeldung.transfer.LoginForm;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ExampleService {
+
+        public boolean fakeAuthenticate(LoginForm lf) {
+                return (lf.getUsername().equals(lf.getUsername()) && lf.getPassword() == lf.getPassword());
```

Comment:
Yeah - it could be simplified - just wanted to flesh that out - here it does qualify as a kind of authentication check since it compares values.